src/transaction/log_recovery_analysis.cpp

```diff
@@ -2415,7 +2512,7 @@ log_recovery_analysis_from_trantable_snapshot (THREAD_ENTRY *thread_p,
     return true;
   });
 
-  LOG_SET_CURRENT_TRAN_INDEX (thread_p, sys_tran_index);
+  log_Gl.mvcc_table.complete_mvccids_if_still_active (LOG_SYSTEM_TRAN_INDEX, in_gaps_mvccids, true);
```

Comment:
Why do we use committed=true here ?
Since the transaction is active and we didn't encounter a "COMMIT" log, I expect the transaction should be considered aborted.

Reply:
Agreed, although from the MVCCID completion point of view does not make a difference. Changed.